ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/ExceptionRequestDetailsPage.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import {
+    Breadcrumb,
+    BreadcrumbItem,
+    DescriptionList,
+    DescriptionListDescription,
+    DescriptionListGroup,
+    DescriptionListTerm,
+    Divider,
+    Flex,
+    FlexItem,
+    PageSection,
+    Text,
+    TextVariants,
+    Title,
+} from '@patternfly/react-core';
+import { useParams } from 'react-router-dom';
+
+import NotFoundMessage from 'Components/NotFoundMessage';
+import { VulnerabilityException } from 'services/VulnerabilityExceptionService';
+import { getDateTime } from 'utils/dateUtils';
+import { exceptionManagementPath } from 'routePaths';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import { vulnerabilityExceptions } from './mockUtils';
+import {
+    RequestExpires,
+    RequestedAction,
+    RequestCreatedAt,
+    RequestScope,
+} from './components/ExceptionRequestTableCells';
+
+import './ExceptionRequestDetailsPage.css';
+
+function getSubtitleText(exception: VulnerabilityException) {
+    switch (exception.exceptionStatus) {
+        case 'PENDING':
+            return 'Pending';
+        case 'DENIED':
+            return `Denied (${exception.cves.length} CVEs)`;
+        case 'APPROVED':
+            return `Approved (${exception.cves.length} CVEs)`;
+        case 'APPROVED_PENDING_UPDATE':
+            return `Approved pending update (${exception.cves.length} CVEs)`;
+        default:
+            return '';
```

Comment:
There is a utility you can return here `ensureExhaustive(exception.exceptionStatus)` the will guarantee that all cases are covered, since the `exceptionStatus` can only be one of the 4 items you have here. If a code change in the future added a new status, or if one of these cases was deleted it would become a compiler error that warns us that we haven't covered all possibilities.